### DISCUSSION TITLE
How to use with postgres?

### DISCUSSION THREAD

**Discussion by yuriysergeev** [burningalchemist/sql_exporter#638] [Q&A] [answered]
hi, there! Sorry, but wanna to ask - for postgres I need to recompile source or HELM already linked to necessary? 
Anyone have installation for Postgres sample? 
I tried install via Helm but always have some issue. 
This is may values.yaml
[values.txt](https://github.com/user-attachments/files/17740105/values.txt)

`sql-exporter-6b74bdb496-4hksg ts=2024-11-13T22:31:26.791Z caller=klog.go:118 level=warn func=Warningf msg="Starting SQL exporter (version=0.16.0, branch=HEAD, revision=a1726ff66775980c1de3ad44bfffb169525b2 │
│ sql-exporter-6b74bdb496-4hksg ts=2024-11-13T22:31:26.792Z caller=klog.go:94 level=debug func=Infof msg="Loading configuration from /etc/sql_exporter/sql_exporter.yml"                                        │
│ sql-exporter-6b74bdb496-4hksg ts=2024-11-13T22:31:26.792Z caller=klog.go:157 level=error func=Fatalf msg="Error creating exporter: no collectors defined for target"   `

thx

**Comment by burningalchemist** [comment, marked as answer]
Hey @yuriysergeev,

PostgreSQL, MySQL and MSSQL drivers are included among some other. What you need is to configure collectors. You can look at the example [here](https://github.com/burningalchemist/sql_exporter/blob/master/helm/ci/postgresql-values.yaml).

Give it a try and let me know, happy to help. 👍
